### ISSUE TITLE
comentario 

### ISSUE THREAD

**Issue by noedavico** [GoldorTeps/Proyecto1Instagram#1]
Intenta que todo el post se visualice en la ventana, que no sea necesario utilizar el scroll lateral.

**Comment by agarzon48** [comment]
Estoy de acuerdo. Voy a rechazar la entrega para tenerla trackeada, pero es un buen trabajo. Simplemente deberías jugar con las dimensiones. Ten en cuenta que has jugado con porcentajes, y por tanto se ve desorbitado en pantallas grandes.

Prueba a combinar tu propuesta con un max-width y cambiar los vw/vh (por ejemplo, en el margin y padding del content) por medidas absolutas (px, rem...). Los márgenes y paddings son uno de los pocos sitios donde podemos jugar sin miedo con este tipo de medidas, y si necesitas hacer correcciones para algún tipo de dispositivo siempre puedes tirar mano de las [media queries](https://developer.mozilla.org/es/docs/Web/CSS/Media_Queries/Using_media_queries).

Revisa eso y avisa por este hilo
